.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,25 @@
+name: Run-Tests
+
+on: [pull_request]
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
```

Comment:
consider using `ubuntu-20.04`
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on

Reply:
I would argue that if ubuntu version matters we have a bug and should fix it

Reply:
We can continue this conversation on MM. Resolving this conversation here for now so I can merge this. Will be happy to make any changes when there is a consensus.